crates/ruff_db/src/diagnostic/render.rs

```diff
@@ -604,7 +605,10 @@ impl<'a> FileResolver<'a> {
 
     /// Returns the path associated with the file given.
     fn path(&self, file: File) -> &'a str {
-        file.path(self.db).as_str()
+        relativize_path(
+            self.db.system().current_directory(),
+            file.path(self.db).as_str(),
+        )
```

Comment:
Sorry for my piece by piece review. 

I assume that doing this here ensures that this change applies to both the full and concise rendering?

I'm asking because that would explain why mypy primer hangs (too large diffs :D)

Reply:
Yes. There are snapshot tests that do cover this.

Good to know about mypy primer hanging. Makes sense, since I think just about ~every diagnostic is impacted by this change.